DeratControl.Ifrastructure/EntitiesConfigurations/OrganizationConfiguration.cs

```diff
@@ -1,10 +1,23 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using DeratControl.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace DeratControl.Infrastructure.EntitiesConfigurations
 {
-    class OrganizationConfiguration
+    class OrganizationConfiguration : IEntityTypeConfiguration<Organization>
     {
+        public void Configure(EntityTypeBuilder<Organization> builder)
+        {
+            builder.HasKey(o => o.Id);
```

Comment:
think of moving common properties of all the entities into separate base entity configuration class

Reply:
ok)

Reply:
Done)

Reply:
sweet)